src/componmnet/PointCloud/render.js

```diff
@@ -6,16 +6,17 @@ import { SelectionBox } from 'three/examples/jsm/interactive/SelectionBox';
 import SelectionHelper from './SelectionHelper';
 
 const updateCamera = (camera, controls, nodeMesh) => {
-    const {x, y, z} = nodeMesh.position;
-   
+    let {x, y, z} = nodeMesh.position;
+    x = parseInt(x);
+    y = parseInt(y);
```

Comment:
Why parse int

Reply:
because the original data is in string format and I found out that if do not parse it to int the camera position will not change

Reply:
We should parse int at the data side not the rendering side